### PR TITLE
Zuri App Bar added

### PR DESCRIPTION
**Pull request by jeremiahseun**
Zuri App Bar implementation into all screens that have app bars.

### CODE REVIEW

File: lib/ui/view/channel/channel_info/widgets/first_section.dart [hunk 1]
```diff
@@ -12,58 +12,58 @@ class FirstSection extends StatelessWidget {
...
-          border: Border.all(width: 1.w, color: AppColors.borderColor),
-          boxShadow: [
+          borderRadius: BorderRadius.circular(6),
+          boxShadow: const [
             BoxShadow(
-              color: Colors.grey.withOpacity(0.5),
-              spreadRadius: 5.r,
-              blurRadius: 6.r,
-              offset: const Offset(0, 3), // changes position of shadow
+              color: Colors.white,
```
Comment (pauleke65) [inline lib/ui/view/channel/channel_info/widgets/first_section.dart:21]: Please make this use a color from the defined AppColors

File: lib/ui/view/channel/channel_view/channel_page_view.dart [hunk 2]
```diff
@@ -76,7 +62,47 @@ class ChannelPageView extends StatelessWidget {
                 ),
               ),
             ],
+            title: "#$channelName",
+            subtitle:
+                "${model.channelMembers.length} member${model.channelMembers.length == 1 ? "" : "s"}",
           ),
+          // AppBar(
```
Comment (pauleke65) [inline lib/ui/view/channel/channel_view/channel_page_view.dart:69]: Remove commented out appbar

**Review by pauleke65 — CHANGES_REQUESTED** [review]
Minor Changes requested

**Comment by YenumJay** [comment]
Great work. Implement changes and we are good.

**Review by YenumJay — APPROVED** [review]
LG

**Review by pauleke65 — APPROVED** [review]
LG

**Pull request merged**